screwdriver/screwdriver_test.go

```diff
@@ -0,0 +1,268 @@
+package screwdriver
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var testDir string = "./testdata"
+
+func validateHeader(tb testing.TB, key, value string, r *http.Request) {
+	tb.Helper()
+	headers, ok := r.Header[key]
+
+	assert.True(tb, ok, fmt.Sprintf("No %s header sent in Screwdriver request", key))
+
+	header := headers[0]
+	assert.Equal(tb, value, header)
+}
+
+func TestNew(t *testing.T) {
```

Comment:
Do we need to test these errors?
https://github.com/screwdriver-cd/sd-local/blob/bfff9dd0772df820d3ac8fb40d93e6acbbe37848/screwdriver/screwdriver.go#L112
https://github.com/screwdriver-cd/sd-local/blob/bfff9dd0772df820d3ac8fb40d93e6acbbe37848/screwdriver/screwdriver.go#L121

Reply:
I fixed.